### PR TITLE
Update to match service behavior

### PR DESCRIPTION
**Pull request by JoshLove-msft**
# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

**Review by lmazuel — APPROVED** [review]
Feels right

File: specification/eventgrid/Azure.Messaging.EventGrid/main.tsp [hunk 1]
```diff
@@ -108,7 +108,7 @@ namespace Azure.Messaging.EventGrid {
     lockToken: LockToken;
```
Comment (l0lawrence) [inline specification/eventgrid/Azure.Messaging.EventGrid/main.tsp:108]: Would this also be a string here: {'lockToken': 'token', 'errorCode': 'BadToken', ....

Comment (JoshLove-msft) [reply specification/eventgrid/Azure.Messaging.EventGrid/main.tsp:108]: Yep, good catch!

Comment (JoshLove-msft) [reply specification/eventgrid/Azure.Messaging.EventGrid/main.tsp:108]: I think this means we can also delete LockToken model.

**Review by lmazuel — APPROVED** [review]

**Pull request merged**